sdf.md

```diff
@@ -980,6 +981,9 @@ present specification.
     with the processes that combine ecosystem and instance specific
     information with an SDF model.
 
+3. `contentType` and `contentCoding` formed a single quality `contentFormat`
```

Comment:
We could reference RFC 9193 here.  We could actually use the ABNF in Section 6 of that spec.  See Appendix A of https://www.ietf.org/archive/id/draft-ietf-core-problem-details-06.html for an example how to do this.